docs/distributed_scenarios.rst

```diff
@@ -0,0 +1,332 @@
+Running distributed scenarios
```

Comment:
I love this documentation Lourens! It's really clear, and necessary as well. I think it is a good idea to test whether someone (me?) can reproduce this. I've openend #84 for that.

Reply:
Yeah. A future me will probably be doing that, and by then I'll have forgotten how this works, so that will be a reasonable simulation, but an actual other person would be even better. Thanks!